### PR TITLE
LURE: Add keymapper support

### PR DESCRIPTION
**Pull request by NabeelShabbir**

### CODE REVIEW

File: engines/lure/fights.cpp [hunk 1]
```diff
@@ -190,26 +190,25 @@ void FightsManager::checkEvents() {
...
+					++actionPtr;
+				if (actionPtr->customAction != kActionNone) {
+					moveNumber = actionPtr->moveNumber;
 					_keyDown = KS_KEYDOWN_1;
 				}
+
 			}
 
-		} else if (events.type() == Common::EVENT_KEYUP) {
+		} else if (events.customType() == Common::EVENT_CUSTOM_ENGINE_ACTION_END) {
```
Comment (OMGPizzaGuy) [inline engines/lure/fights.cpp:211]: I believe this was a mistake - the event type is not same as the event custom type, correct?

Comment (NabeelShabbir) [reply engines/lure/fights.cpp]: Fixed.

**Review by OMGPizzaGuy — APPROVED** [review]
This looks good to me!

**Comment by bluegr** [comment]
Looks good, thanks!

**Pull request merged**